templates/alertmanagerconfig.go

```diff
@@ -31,17 +31,43 @@ func convertToApiExtV1JSON(val interface{}) apiextensionsv1.JSON {
 	return out
 }
 
+var _true = true
+
+var smtpRegxRuleMatchers = "CephClusterNearFull" +
+	"|CephClusterCriticallyFull" +
+	"|CephClusterReadOnly" +
+	"|CephOSDCriticallyFull" +
+	"|CephOSDNearFull" +
+	"|PersistentVolumeUsageNearFull" +
+	"|PersistentVolumeUsageCritical"
+
+var pagerdutyRegxRuleMatchers = "critical|warning"
```

Comment:
Why aren't we using exact alerts names here?

Reply:
matchers are ANDed and with regex are ORed. Thus alertnames are provided as regex.